src/main/kotlin/io/libp2p/security/noise/NoiseXXSecureChannel.kt

```diff
@@ -33,9 +34,11 @@ private enum class Role(val intVal: Int) { INIT(HandshakeState.INITIATOR), RESP(
 
 private val log = LogManager.getLogger(NoiseXXSecureChannel::class.java)
 private const val HandshakeNettyHandlerName = "HandshakeNettyHandler"
+private const val NoiseCodeNettyHandlerName = "NoiseXXCodec"
+private const val MaxCipheredPacketLength = 65535
 
 class UShortLengthCodec : CombinedChannelDuplexHandler<LengthFieldBasedFrameDecoder, LengthFieldPrepender>(
-    LengthFieldBasedFrameDecoder(0xFFFF, 0, 2, 0, 2),
+    LengthFieldBasedFrameDecoder(0xFFFF + 2, 0, 2, 0, 2),
```

Comment:
Is the frame length coming from the constant above?:
```suggestion
    LengthFieldBasedFrameDecoder(MaxCipheredPacketLength + 2, 0, 2, 0, 2),
```